python/circuitsvis/attention.py

```diff
@@ -1,13 +1,25 @@
 """Attention visualisations"""
-from typing import List, Optional, Union
-
+import os
+import uuid
+import json
+import einops
+import random
+import itertools
+import webbrowser
+import torch as t
 import numpy as np
-import torch
+from torch import Tensor
+from pathlib import Path
+from jaxtyping import Float
+from collections import defaultdict
+from IPython.display import display, HTML, Javascript
+from typing import List, Optional, Union, Tuple, Literal
 from circuitsvis.utils.render import RenderedHTML, render
+from transformer_lens import ActivationCache, utils, HookedTransformer, HookedTransformerConfig
 
 
 def attention_heads(
-    attention: Union[list, np.ndarray, torch.Tensor],
+    attention: Union[list, np.ndarray, t.Tensor],
```

Comment:
all the .py files here use the torch.Tensor convention, lets be consistent first and think about changing the convention everywhere second